README.md

```diff
@@ -13,6 +13,9 @@ Audience: Node.js developers, CS students, security enthusiasts
 
 ## Getting started
 
+247ea66c6d2d10d12f11af72cde08aef6f1800ec
```

Comment:
If this is a GitHub API key, you should revoke it (if it's just randomness you generated for testing crypto that's okay).